### ISSUE TITLE
How to get the clear edge

### ISSUE THREAD

**Issue by ValentiaSulli** [intel-isl/MiDaS#20]
Hi，thank you for your work！
I trained the model following your job.  I found the released model has more clear object-edges. Which part does benefit the clear edges? 
Thanks in advance!

**Comment by ranftlr** [comment]
There are multiple things contributing to sharpness of the results:

- Relatively high resolution of data during training and testing. This is a major difference between MiDaS v1 and MiDaS v2 and improved sharpness considerably.
- Sharpness of the ground truth inverse depth. If your training data isn't sharp, predictions won't be. 
- The gradient matching term (equation 9 in the paper) has a noticeably positive effect. See [32] for example comparisons with and without this term.

**Comment by ranftlr** [comment]
Closing due to inactivity

**Comment by oak-JHCheng** [comment]
> Hi，thank you for your work！ I trained the model following your job. I found the released model has more clear object-edges. Which part does benefit the clear edges? Thanks in advance!

Hi, I'm following this job(DPT) now. I write a train code by myself but it only applies to NYUv2 dataset(absolute depth). I don't know about the exact code implementation of disparity and mixing strategy. So can you share your training code with me? Thank you in advance!

**Comment by ValentiaSulli** [comment]
已收到您的邮件，谢谢
